gometalinter/def.bzl

```diff
@@ -8,26 +9,30 @@ def _gometalinter_impl(ctx):
         args.append("--no-config")
     args.extend(ctx.attr.paths)
     out_file = ctx.actions.declare_file(ctx.label.name + ".bash")
+    sdk = ctx.attr._go_sdk[GoSDK]
     substitutions = {
+        "@@GOMETALINTER_SHORT_PATH@@": shell.quote(ctx.executable._gometalinter.short_path),
+        "@@ARGS@@": shell.array_literal(args),
         "@@PREFIX_DIR_PATH@@": shell.quote(paths.dirname(ctx.attr.prefix)),
         "@@PREFIX_BASE_NAME@@": shell.quote(paths.basename(ctx.attr.prefix)),
-        "@@ARGS@@": shell.array_literal(args),
-        "@@GOMETALINTER_SHORT_PATH@@": shell.quote(ctx.executable._gometalinter.short_path),
+        "@@NEW_GOROOT@@": shell.quote(sdk.root_file.dirname),
     }
     ctx.actions.expand_template(
         template = ctx.file._runner,
         output = out_file,
         substitutions = substitutions,
         is_executable = True,
     )
-    transitive_depsets = []
+    transitive_depsets = [
+        depset(sdk.srcs),
+        depset(sdk.tools),
+    ]
     default_runfiles = ctx.attr._gometalinter[DefaultInfo].default_runfiles
     if default_runfiles != None:
         transitive_depsets.append(default_runfiles.files)
 
     runfiles = ctx.runfiles(
-        files = ctx.files._sdk_files,
```

Comment:
Benefit 1 of this PR: we now only depend on `src` and `tools`, not all files from SDK. Less symlinks to create.

Reply:
Since you're here: https://github.com/bazelbuild/bazel-gazelle/issues/259 we might be able to get rid of some of the wrapping bash scipts